students/biegon_piotr/lesson_05_list_comprehensions/numbers_filter.py

```diff
@@ -0,0 +1,14 @@
+def filter_list(int_list, int_range):
+    out_list = []
+    for x in int_list:
+        if x not in int_range:
+            out_list.append(x)
+    return out_list
```

Comment:
it's okay, but ideally this whole function would be replaced by single list comprehension